### PR TITLE
refactor: update img source blog/2020

### PR DESCRIPTION
**Pull request by Ylikj**
update img source blog/2020

### CODE REVIEW

File: website/blog/2020/12/11/beike-how-to-build-gateway-based-on-apache-apisix.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ tags: [User Case]
 
 ## 选择 Kong 还是 Apache APISIX
 
-![Apache APISIX vs Kong in QPS](https://static.apiseven.com/2020/05/1588752135-Snipaste_2020-05-06_16-02-04.png)
-![Apache APISIX 架构图](https://static.apiseven.com/2020/05/1588756665-Snipaste_2020-05-06_17-17-29.png)
-![Apache APISIX 技术亮点](https://static.apiseven.com/2020/05/1588756618-Snipaste_2020-05-06_17-16-13.png)
+![Apache APISIX vs Kong in QPS](https://static.apiseven.com/202108/1636562127625-57da7a81-3ce8-4ffd-9baa-7e8d901430f6.png)
```
Comment (juzhiyuan) [inline website/blog/2020/12/11/beike-how-to-build-gateway-based-on-apache-apisix.md:22]: Hi, may I know what this PR does?

Comment (Ylikj) [reply website/blog/2020/12/11/beike-how-to-build-gateway-based-on-apache-apisix.md:22]: According to the suggestion of #694, the pictures in the blog folder that need to be converted to cdn are pulled in different prs according to the month, which is convenient for review@juzhiyuan

Comment (juzhiyuan) [reply website/blog/2020/12/11/beike-how-to-build-gateway-based-on-apache-apisix.md:22]: 1. Please use English in public;
2. Please attach the Related Issue/PR/Comment Link with this issue.

**Comment by juzhiyuan** [comment]
Hi, I'm going to close this PR because of too many conflicts 😂

**Comment by yzeng25** [comment]
> Hi, I'm going to close this PR because of too many conflicts 😂

ok

**Pull request closed without merging**